numba/core/utils.py

```diff
@@ -146,7 +75,8 @@ def safe_relpath(path, start=os.curdir):
     'is': operator.is_,
     'is not': operator.is_not,
     # This one has its args reversed!
-    'in': operator.contains
+    'in': operator.contains,
+    '@': operator.matmul,
```

Comment:
Are you planning to add support for the `@` operator on numpy arrays (probably not in this PR)?

Reply:
No, I just moved this from a line further down this file.

Reply:
`@` operator already works on NumPy arrays IIRC.
```pycon
In [1]: from numba import njit

In [2]: import numpy as np

In [3]: a = np.ones((3, 3))

In [4]: @njit
   ...: def foo(a):
   ...:     return a@a
   ...: 

In [5]: foo(a)
Out[5]: 
array([[3., 3., 3.],
       [3., 3., 3.],
       [3., 3., 3.]])
```